packages/config/config/devices/0x0346/panic_button_gen2.json

```diff
@@ -0,0 +1,71 @@
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Ring Alarm Panic Button",
+	"description": "Panic Button (2nd Gen)",
+	"devices": [
+		{
+			"productType": "0x0801",
+			"productId": "0x0401"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Heartbeats",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 1,
+			"maxValue": 70,
+			"defaultValue": 70
+		},
+		{
+			"#": "2",
+			"label": "Number of Application Level Retries",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1
+		},
+		{
+			"#": "3",
+			"label": "Application Level Retry Base Wait Time Period",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 60,
+			"defaultValue": 5
+		},
+		{
+			"#": "4",
+			"label": "Button Hold-Time",
+			"valueSize": 1,
+			"unit": "0.5 seconds",
+			"minValue": 1,
+			"maxValue": 20,
+			"defaultValue": 6
+		},
+		{
+			"#": "5",
+			"label": "Wait Time for A Supervisory Report Response",
```

Comment:
```suggestion
			"label": "Supervision Report Timeout",
			"description": "How long to wait for a Supervision Report response to a Supervision Get",
```

Reply:
Done FYI- I picked up these labels from the Gen 1 Ring Panic button on the zwa website.  Deleted the model number and meta data because device wasn't a match and could cause confusion.